rails/7/gnarly.rb

```diff
@@ -345,12 +336,12 @@ def post_install_instructions
 end
 
 def format_ruby
-  run 'bin/rubocop -A --fail-level error'
+  run 'bundle exec rubocop -A --fail-level error'
 end
 
 def completion_notification
   puts ''
-  ascii_art
+  puts ascii_art
   post_install_instructions
```

Comment:
I'd argue that post_install_instructions should be one big string that gets puts-ed here.